src/helpers/TextFormatting.js

```diff
@@ -447,7 +447,7 @@ export function t(key, options) {
     let val = i18next.t(key, options);
     if (!val) {
         let opts = options || {};
-        opts.lng = 'en-us';
+        opts.lng = 'dev';
```

Comment:
You would never want to show the dev language in production. It falls back to en-us as that is the default translation that we know will always be shipped complete